Sources/Instrumentation/MultiplexInstrument.swift

```diff
@@ -1,25 +1,35 @@
 import Baggage
 
-/// A pseudo-`Instrument` that may be used to instrument using multiple other `AnyInstrument`s
-/// across a common `BaggageContext`.
-public struct MultiplexInstrument<InjectInto, ExtractFrom> {
-    private var instruments: [AnyInstrument<InjectInto, ExtractFrom>]
+/// A pseudo-`Instrument` that may be used to instrument using multiple other `Instrument`s across a
+/// common `BaggageContext`.
+public struct MultiplexInstrument {
+    private var instruments: [Instrument]
```

Comment:
great, no longer fixed to specific Into/Froms.

Reply:
Yep, way nicer to work with now 👍